--- origsrc/lensfun-0.2.5/tools/makedep/main.cpp	2008-10-25 14:24:28.000000000 -0500
+++ src/lensfun-0.2.5/tools/makedep/main.cpp	2011-09-20 13:58:16.086011100 -0500
@@ -130,7 +130,7 @@ sighandler (int sig)
     fatalerr ("got signal %d\n", sig);
 }
 
-#if defined(USG) || defined(_WIN32) || defined(OS_OS2) || defined(OS_NEXT) || defined(OS_BE) || defined (OS_DOS) || defined (OS_SOLARIS)
+#if defined(USG) || defined(_WIN32) || defined(OS_OS2) || defined(OS_NEXT) || defined(OS_BE) || defined (OS_DOS) || defined (OS_SOLARIS) || defined(__CYGWIN__)
 #define USGISH
 #endif
 
